src/main/java/project/cinema/dao/UserDao.java

```diff
@@ -7,4 +7,6 @@
     User add(User user);
 
     Optional<User> findByEmail(String email);
+
+    User getById(Long id);
```

Comment:
```suggestion
    Optional<User> getById(Long id);
```
let's return Optional on dao layer - and User on service layer